### PR TITLE
xdr: Update XDR to meet core's fees changes

### PR DESCRIPTION
**Pull request by tsachiherman**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://developers.stellar.org/api/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Update XDR to meet core's fees changes

### Why

We need to compile against core.

### Known limitations

N/A

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,10 +10,11 @@ xdr/Stellar-types.x \
 xdr/Stellar-contract-env-meta.x \
 xdr/Stellar-contract-spec.x \
 xdr/Stellar-contract.x \
-xdr/Stellar-internal.x
+xdr/Stellar-internal.x \
+xdr/Stellar-contract-config-setting.x
 
 XDRGEN_COMMIT=80e38ef2a96489f6b501d4db3a350406e5aa3bab
-XDRNEXT_COMMIT=d01024346cc0a23e155b9cbf4dbf1780adf3890d
+XDRNEXT_COMMIT=95e988b896181d28055d61cc10a95448fe6f04b2
```
Comment (sreuland) [inline Makefile:17]: which commit on stellar-xdr:next does this belong to, was expecting to see the commit on that branch:

https://github.com/stellar/stellar-xdr/commits/next

is the latest commit on stellar-xdr:next equivalent for reference?

Comment (tsachiherman) [reply Makefile:17]: It wasn't merged the last time I checked. ( I.e. see dima's pr )

File: services/horizon/internal/ingest/processors/operations_processor.go [hunk 2]
```diff
@@ -631,65 +631,67 @@ func (operation *transactionOperationWrapper) Details() (map[string]interface{},
...
-				serializedParam := map[string]string{}
-				serializedParam["value"] = "n/a"
-				serializedParam["type"] = "n/a"
-
-				if name, ok := param.ArmForSwitch(int32(param.Type)); ok {
-					serializedParam["type"] = name
-					if raw, err := param.MarshalBinary(); err == nil {
-						serializedParam["value"] = base64.StdEncoding.EncodeToString(raw)
+		for _, function := range op.Functions {
+			details["function"] = function.Args.Type.String()
```
Comment (sreuland) [inline services/horizon/internal/ingest/processors/operations_processor.go:635]: I think we need to update the horizon protocol model, `InvokeHostFunction` to capture multi-function:

https://github.com/tsachiherman/stellar-go/blob/tsachi/xdr-update-2/protocols/horizon/operations/main.go#L362

something like:

```
type ContractFunction {
        Parameters          []HostFunctionParameter      `json:"parameters"`
	Function            string                       `json:"function"`
}

type InvokeHostFunction struct {
	Base
         Functions.       []ContractFunction
	Footprint           string                       `json:"footprint"`
        AssetBalanceChanges []AssetContractBalanceChange `json:"asset_balance_changes"`
}
```

File: Makefile (hunk 1 above)
Comment (tsachiherman) [reply Makefile]: I've synced up with the latest xdr.

File: services/horizon/internal/integration/sac_test.go [hunk 3]
```diff
@@ -1112,26 +1127,34 @@ func burn(itest *integration.Test, sourceAccount string, asset xdr.Asset, assetA
...
+		V: 1,
+		SorobanData: &xdr.SorobanTransactionData{
+			Resources: xdr.SorobanResources{
+				Instructions:              0,
+				ReadBytes:                 0,
+				WriteBytes:                0,
+				ExtendedMetaDataSizeBytes: 0,
+				Footprint: xdr.LedgerFootprint{
+					ReadOnly: []xdr.LedgerKey{
+						// TODO: derive the needed value
```
Comment (sreuland) [inline services/horizon/internal/integration/sac_test.go:1141]: @tsachiherman , I think pr is ready to merge, i had to disable the tests in sac_test.go as they all depended on the preflight http call to core which is gone now, the tests can be re-enabled on later iteration when time to decide how to derive the footprint inline here. 

there are working contract tx sub tests in invokehostfunction_test.go which derive the needed footprint for those that can be used as reference, but those are non-sac, so the footprint may not be same approach.

**Review by tsachiherman — COMMENTED** [review]
looks good. Unfortunately, since I'm the one who created the PR, I'm unable to approve it.

**Pull request merged**